### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arampuria19**
I have added a new section for the enthusiasts who are looking forward to begin their journey in the world of open source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,9 @@ This is a list of resources for engineering students. This list is maintained by
 5. [C++ STL by Utkarsh gupta ](https://youtu.be/PZogbfU4X5E)
 6. [C++ STL(Full Course) by Luv ](https://www.youtube.com/playlist?list=PLauivoElc3gh3RCiQA82MDI-gJfXQQVnn)
 
+## Open Source Resources <a name="open-sourcen"></a>
```
Comment (aaryan2134) [inline README.md:47]: Correct the linking please here
Refer to other headings

File: README.md [hunk 2]
```diff
@@ -44,3 +44,9 @@ This is a list of resources for engineering students. This list is maintained by
...
+1. [Roadmap for beginners](https://towardsdatascience.com/getting-started-with-open-source-roadmap-35c6f76682b7)
+2. [Beginner's Course on Git & Github](https://youtu.be/RGOj5yH7evk).
+3. [Git & GitHub Tutorial For Beginners (Hindi)](https://www.youtube.com/watch?v=gwWKnnCMQ5c&ab_channel=CodeWithHarry) (Youtube Channel).
+4. [Git Documentation](https://git-scm.com/doc).
```
Comment (aaryan2134) [inline README.md:51]: Please add something else here. 4th and 5th are almost same

**Review by aaryan2134 — CHANGES_REQUESTED** [review]
Please add this section in the index and make the suggested changes

File: README.md (hunk 2 above)
Comment (arampuria19) [reply README.md:51]: yeah sure! i'll do it

**Comment by aaryan2134** [comment]
Are you still working on this? or should I close this?

**Comment by arampuria19** [comment]
Yeah! I'll make the asked changes by tonight. Got delayed coz of some unavoidable circumstances.

**Comment by arampuria19** [comment]
@aaryan2134 done with all asked changes

**Comment by aaryan2134** [comment]
Great!
Please resolve conflicts for merging

**Comment by arampuria19** [comment]
resolved!

**Comment by aaryan2134** [comment]
Conflicts are still there. Please look into it again

**Comment by arampuria19** [comment]
afaik I have resolved both the conflicts created. If still it's making an issue then shall I put forth a new PR and close this one !?

**Pull request closed without merging**